samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/AssessmentGradingFacadeQueries.java

```diff
@@ -3009,7 +3009,7 @@ public int autoSubmitAssessments() {
 						" order by a.publishedAssessmentId, a.agentId, a.forGrade desc, a.assessmentGradingId");
 	    
 		query.setTimestamp("currentTime",currentTime);
-		query.setParameterList("status", new ArrayList<Integer>( Arrays.asList(AssessmentGradingData.REMOVED, AssessmentGradingData.IN_PROGRESS, AssessmentGradingData.NO_SUBMISSION ) ));
+		query.setParameterList("status", new ArrayList<Integer>( Arrays.asList(AssessmentGradingData.REMOVED, AssessmentGradingData.NO_SUBMISSION ) ));
```

Comment:
Do you need the new ArrayList bit? Arrays.asList will returned a typed list anyway.

Reply:
good point